Topic2/src/main/java/Blogcase/Entry.java

```diff
@@ -0,0 +1,67 @@
+package Blogcase;
+
+import java.text.DateFormat;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+public class Entry {
+
+	private int entryID;
+	private String title;
+	private String director;
+	private String genero;
+	private String postDate;
+	private static int IDGenerator = 0;
+
+	public Entry(String title, String director, String genero, String postDate) {
+		this.entryID = IDGenerator++;
```

Comment:
Be careful with this ID generation method, this will work only if you have 1 instance running and never save the objects, you can use UUID when you need unique IDs